andela/wg-aces
# #159250154 Add hover effect on muscles to show exercises

### What does this PR do?
A user should be able to hover the cursor over a body part and get exercises for that body part

### Description of Task to be completed?
Currently a user can only view excercises corresponding to the muscle links they hover over. However, the user should also be able to view the excercises of the muscle image.

### How should this be manually tested?
Log in to wger
Navigate to `Exercises` dropdown
Click `Muscle overview`
Hover over the muscles on the body parts to see the exercises

### What are the relevant pivotal tracker stories?
[#159250154](https://www.pivotaltracker.com/story/show/159250154)

[Deliver #159250154]

### Screenshots
![image](https://user-images.githubusercontent.com/30747298/44804864-4f2f2280-abcb-11e8-8961-faf7bac73936.png)
